ContentBlocks/ContentElements/carousel/templates/backend-preview.html

```diff
@@ -0,0 +1,12 @@
+<f:asset.css identifier="content-block-t3docs-t3docs/carousel-be" href="{cb:assetPath()}/preview.css"/>
+
+<div class="row">
```

Comment:
We recently merged a new feature for backend previews, which requires now to add a Layout (Not released yet): https://docs.typo3.org/p/friendsoftypo3/content-blocks/main/en-us/ChangeLog/1.1/Index.html

We plan to release it together with TYPO3 13.2

Reply:
Ok, will change that once 1.1 is released. You are still making breaking changes then?

Reply:
@linawolf This is not breaking, only a deprecation.

Reply:
IC